libraries/AP_Soaring/AP_Soaring.h

```diff
@@ -65,34 +71,61 @@ class SoaringController {
     AP_Float alt_max;
     AP_Float alt_min;
     AP_Float alt_cutoff;
+    AP_Float max_drift;
 
 public:
     SoaringController(AP_AHRS &ahrs, AP_SpdHgtControl &spdHgt, const AP_Vehicle::FixedWing &parms);
 
+    typedef enum LoiterStatus {
```

Comment:
We prefer `enum class` now-adays to avoid namespace conflicts.

Reply:
Refactored to use enum class